cmd/boulder-ca/main.go

```diff
@@ -195,6 +195,12 @@ func main() {
 		issuers = append(issuers, issuer)
 	}
 
+	// Output issuer details here rather than in Issuers loop above to avoid
+	// interleaving with clock logs.
```

Comment:
What are the clock logs this would interleave with?

Reply:
I assume this is "sometimes cmd.Clock() prints output, if it's being set from an environment variable"?

In that case, let's just move line 232 up towards the top of this function, and pass `clk` instead of `cmd.Clock()` into `LoadIssuer()`.